repairnator/src/main/java/fr/inria/spirals/repairnator/process/step/CheckoutRepository.java

```diff
@@ -0,0 +1,82 @@
+package fr.inria.spirals.repairnator.process.step;
+
+import fr.inria.spirals.jtravis.entities.Build;
+import fr.inria.spirals.jtravis.entities.PRInformation;
+import fr.inria.spirals.repairnator.process.git.GitHelper;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Created by fernanda on 02/03/17.
+ */
+public class CheckoutRepository extends AbstractStep {
```

Comment:
ok so this one should be abstract then: you don't want to execute the step as it.